src/backend/distributed/planner/query_pushdown_planning.c

```diff
@@ -549,23 +544,6 @@ SubqueryMultiNodeTree(Query *originalQuery, Query *queryTree,
 	else if (subqueryPushdownError)
 	{
 		RaiseDeferredErrorInternal(subqueryPushdownError, ERROR);
```

Comment:
Using RaiseDeferredError() could make more sense so that we could tell compiler that rest of the code is unreachable.

```c
#define RaiseDeferredError(error, elevel) \
	do { \
		RaiseDeferredErrorInternal(error, elevel); \
		if (__builtin_constant_p(elevel) && (elevel) >= ERROR) { \
			pg_unreachable(); } \
	} \
	while (0)
```